OrbitBase/include/OrbitBase/Logging.h

```diff
@@ -33,9 +33,11 @@ constexpr const char* kLogTimeFormat = "%Y-%m-%dT%H:%M:%E6S";
     if (path_it__ != path__.begin()) file__ = (--path_it__)->string();                       \
     if (path_it__ != path__.begin()) dir__ = (--path_it__)->string();                        \
     std::string file_and_line__ = absl::StrFormat("%s/%s:%d", dir__, file__, __LINE__);      \
+    if (file_and_line__.size() > 32)                                                         \
+      file_and_line__ = file_and_line__.substr(file_and_line__.size() - 32);                 \
```

Comment:
I would rather see full names than perfect formatting.

Reply:
So you would want to keep `%28s` or should it be changed to `%s`?

In any case, in my experience reading the logs right now is super annoying. We can increase 32 to an even higher number, but without a fixed initial column the format is all over the place.

Reply:
I think I prefer %s there, seems like otherwise we are wasting space on the screen...